pom.xml

```diff
@@ -9,16 +9,16 @@
 	<description>Kafka Sink Connector for RDF update streaming to GraphDB</description>
 
 	<properties>
-		<graphdb.version>10.0.0-RDF4J4-BOOTSTRAP1</graphdb.version>
-        <kafka.version>2.8.0</kafka.version>
-		<slf4j.version>1.7.30</slf4j.version>
+		<graphdb.version>10.5.0</graphdb.version>
```

Comment:
Is this property used? I don't see any references in the code

Reply:
Addressed